tests/ui/cast_size_32bit.stderr

```diff
@@ -1,118 +1,183 @@
-error: casting `isize` to `i8` may truncate the value
+warning: casting `isize` to `i8` may truncate the value
```

Comment:
A 32bit only test that needs manually reverting

Reply:
I went through all `//@only` and `//@ignore` tests and fixed them.